src/IndexingWrapper.cpp

```diff
@@ -243,6 +245,71 @@ indexWrapper(Rcpp::CharacterVector & dbpath, Rcpp::DataFrame & dataFrame, Rcpp::
 	    }
 	}
 
+	//Add value slots
+	for (Rcpp::List::iterator it = slotList.begin(); it != slotList.end(); ++it) {
+	    Rcpp::List list = it.operator*();
+	    bool serialiseVal = false;
+	    bool dfVal = true; // use original row values in data frame
+	    int slot;
+	    std::string val;
+
+	    try {
+		slot = Rcpp::as<int>(list["slot"]);
+	    } catch (...) {
+		Rcpp::stop("Slot number should be specified.");
```

Comment:
Will we see this if they provide `list["slot"]` but it isn't parseable as an integer? That should probably get a different error.